diff --git a/verilog/src/unisims/MMCME2_ADV.v b/verilog/src/unisims/MMCME2_ADV.v
index 7b08f56..3eb57ba 100644
--- a/verilog/src/unisims/MMCME2_ADV.v
+++ b/verilog/src/unisims/MMCME2_ADV.v
@@ -4509,7 +4509,7 @@ endfunction
     $width (posedge PWRDWN, 0:0:0, 0, notifier);
     $width (posedge RST, 0:0:0, 0, notifier);
 `endif
-    specparam PATHPULSE$ = 0;
+    specparam PATHPULSE_ = 0;
   endspecify
 `endif
 endmodule
